boxoffice/models/line_item.py

```diff
@@ -325,6 +332,11 @@ def get_confirmed_line_items(self):
 Order.get_confirmed_line_items = property(get_confirmed_line_items)
 
 
+def initial_line_items(self):
+    return LineItem.query.filter(LineItem.order == self, LineItem.previous == None, LineItem.status.in_(LINE_ITEM_STATUS.TRANSACTION))
+Order.initial_line_items = property(initial_line_items)
```

Comment:
Both this and the other function could be lazy relationships with elaborate `primaryjoin` conditions. Seems cleaner.